Examples/ArcGISToolkitExamples/JobManagerExample.swift

```diff
@@ -135,20 +136,11 @@ class JobManagerExample: TableViewController {
         toolbar.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
         toolbar.heightAnchor.constraint(equalToConstant: toolbarHeight).isActive = true
         
-        if #available(iOS 11.0, *) {
-            // move safe area up above toolbar
-            // (this adjusts tableview contentInsets to correctly scroll behind toolbar)
-            additionalSafeAreaInsets = UIEdgeInsetsMake(0, 0, toolbarHeight, 0)
-            // now anchor toolbar below new safe area
-            toolbar.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor).isActive = true
-        }
-        else {
-            // pre-iOS 11, adjust content inset of tableview to go under toolbar
-            tableView.contentInset = UIEdgeInsetsMake(0, 0, toolbarHeight, 0)
-            tableView.scrollIndicatorInsets = tableView.contentInset
-            // anchor toolbar to bottom of view
-            toolbar.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
-        }
+        // move safe area up above toolbar
+        // (this adjusts tableview contentInsets to correctly scroll behind toolbar)
+        additionalSafeAreaInsets = UIEdgeInsets.init(top: 0, left: 0, bottom: toolbarHeight, right: 0)
```

Comment:
I wasn't sure why xcode wrote that like that... I thought it was odd too. A remnant of the swift 4.2 conversion.

Reply:
I recall that Xcode used to have a hard time autocompleting initializers after typing the opening parenthesis (`UIEdgeInsets(`). The workaround was to type the dot (`UIEdgeInsets.`) and then it would list the available initializers. Autocompleting one of those suggesting would leave the `.init` portion.